config/alphagov.yml

```diff
@@ -3,6 +3,10 @@
 
 common_properties: &common_properties
   security_alerts: true
+  morning_seal_quotes: false
```

Comment:
Can this second commit be fixed up to the first?
I can see why you've split it out - to make it easier to review - but otherwise the `morning_seal_quotes: env["MORNING_SEAL_QUOTES"] == "true" || config["morning_seal_quotes"],` bit in the first commit doesn't really work (as `config["morning_seal_quotes"]` would be undefined at the time of commit).

Also makes it a bit easier to determine whether we've carried over the arrays of team names from the scripts, into this config.

Reply:
Fair enough, I thought it would be easier to review with the YAML in a separate commit but you're right.